### PR TITLE
[azure] add ci job for azure-stack-ipi-proxy

### PR DESCRIPTION
**Pull request by jinyunma**

### CODE REVIEW

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-stack-ipi-proxy-p3-f28

**Comment by jinyunma** [comment]
cc @jianlinliu @liangxia to review

Job [periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-stack-ipi-proxy-p3-f28](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/36743/rehearse-36743-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-stack-ipi-proxy-p3-f28/1630148894472015872) failed due to e2e failure cases.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -1746,6 +1746,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-stack-ipi
+- as: azure-stack-ipi-proxy-p3-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1749]: How about add one more job for 4.12

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1749]: okay, will change to p2.
Bug [2104997](https://bugzilla.redhat.com/show_bug.cgi?id=2104997) is only fixed on 4.13 now, and not backport yet. Will add 4.12 job when bug get fixed.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml]: ah. got it

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 2]
```diff
@@ -1746,6 +1746,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-stack-ipi
+- as: azure-stack-ipi-proxy-p2-f7
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1749]: Maybe we can lower test frequency of `azure-stack-ipi` a bit to `f28` based on `azure-stack-ipi-proxy` almost can cover the common ipi install.

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1749]: agree, updated.

**Pull request merged**